fast/stages/2-networking-c-nva/main.tf

```diff
@@ -18,6 +18,7 @@
 
 locals {
   custom_roles = coalesce(var.custom_roles, {})
+  nva_zones    = ["b", "c"]
```

Comment:
should this be a variable?

Reply:
I thought about it, and my conclusion was _no because_ all the NVA-related code is bound to be replaced when NVAs will be replaced, so I kept their variable surface to zero (see also NCC ASNs).

Reply:
agreed